{"name": "test_loginerror_po", "status": "passed", "attachments": [{"name": "stdout", "source": "482813b9-a038-4114-949d-32641e2eaf09-attachment.txt", "type": "text/plain"}], "start": 1660206892294, "stop": 1660206910678, "uuid": "11761e1a-e155-4b99-bcb0-bf087388498b", "historyId": "0b385e0e45891bc308040f7b2d6c9afd", "testCaseId": "a5645c28040a29df8d8a8de870a9df21", "fullName": "testcase.test_demo.TestDemo#test_loginerror_po", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_demo"}, {"name": "subSuite", "value": "TestDemo"}, {"name": "host", "value": "LAPTOP-68T4BLK3"}, {"name": "thread", "value": "18104-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_demo"}]}